### PR TITLE
Update Contact Details 2020-2021

### PR DESCRIPTION
**Pull request by Pramodh-G**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -18,6 +18,28 @@ facebook_username:  pclubiitk
 disqus_id: 
 
 excoordies:
+  '2019 - 20':
+    'Aditya Gulati':
+
```
Comment (dryairship) [inline _config.yml]: remove the blank lines

File: contact.html [hunk 2]
```diff
@@ -3,129 +3,153 @@
...
-  'majorwar':
-    name: Nilay Majorwar
-    github: NilayMajorwar
-    fb: nilay.majorwar
-    number: 9559084545
-    hall: 2
-  'darshi':
-    name: Priydarshi Singh
+    image: deepankur.jpg
+  'Priydarshi Singh':
```
Comment (dryairship) [inline contact.html]: Alphabetical order

File: contact.html [hunk 3]
```diff
@@ -3,129 +3,153 @@
...
-    github: sartkap
-    fb: sarthak.kapoor.007
-    number: 9910175150
-    hall: 5
-  'tanvin':
-    name: Tanvi Ajay Nerkar
-    github: tanvinerkar
-    fb: tanvi.nerkar.5
-    number: 9916304234
+    image: priyadarshi.jpg
```
Comment (dryairship) [inline contact.html]: Spelling!

**Comment by adarshaj** [comment]
Congrats to new coordis! 🎉

**Comment by dryairship** [comment]
> Congrats to new coordis! :tada: 

Thank you @adarshaj !

**Pull request merged**